Libraries/WebSocket/RCTSRWebSocket.m

```diff
@@ -14,9 +14,10 @@
 //   limitations under the License.
 //
 
-#if !TARGET_OS_OSX // TODO(macOS ISS#2323203)
+#import <TargetConditionals.h> // [TODO(macOS ISS#2323203)
+#if !TARGET_OS_OSX
 #import <Endian.h>
-#endif // TODO(macOS ISS#2323203)b
+#endif // ]TODO(macOS ISS#2323203)
```

Comment:
it seems like there are extra brackets in the comments?

Reply:
That's the closing brace from the comment above